src/app/shared/entity/cluster.ts

```diff
@@ -218,6 +218,7 @@ export class ClusterNetwork {
   pods?: NetworkRanges;
   proxyMode?: ProxyMode;
   services?: NetworkRanges;
+  KonnectivityEnabled?: boolean;
```

Comment:
Does it really start with the upper case letter in the API?

Reply:
Oops not at all. Fixing.